src/metabase/driver/athena.clj

```diff
@@ -127,23 +138,65 @@
                             database-type))
           :type/*)))
 
+(defn- run-query
+  "Workaround for avoiding the usage of 'advance' jdbc feature that are not implemented by the driver yet.
+   Such as prepare statement"
+  [database query]
+  (log/infof "Running Athena query : '%s'..." query)
+  (try
+    (jdbc/query (sql-jdbc.conn/db->pooled-connection-spec database) (string/replace query ";" " ") {:raw? true})
```

Comment:
The `replace query ";" " "` function is useful for native SQL queries as well. Is there a way to use this function for those?